object/attribute.go

```diff
@@ -5,69 +5,61 @@ import (
 )
 
 // Attribute represents v2-compatible object attribute.
```

Comment:
```suggestion
// Attribute represents object attribute.
```

Reply:
why? thought all public types should have such info